docs/src/content/docs/da/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Starlight 🌟 Byg dokumentationssider med Astro
+description: Starlight helps you build beautiful, high-performance documentation websites with Astro.
+template: splash
```

Comment:
We should include these extra properties like the English index page here to avoid an edit link at the bottom of the homepage:

```suggestion
template: splash
editUrl: false
lastUpdated: false
```